packages/solutions/app-tools/src/analyze/getBundleEntry.ts

```diff
@@ -24,6 +24,16 @@ const ensureExtensions = (file: string) => {
  */
 const isDirectory = (file: string) => !path.extname(file);
 
+const isSubDirOrEqual = (parent: string, child: string): boolean => {
```

Comment:
Can we add some unit tests for this method~

Reply:
done~